tasmota/include/tasmota_template.h

```diff
@@ -54,6 +54,7 @@ enum UserSelectablePins {
   GPIO_SDM630_TX, GPIO_SDM630_RX,      // SDM630 Serial interface
   GPIO_TM1638CLK, GPIO_TM1638DIO, GPIO_TM1638STB,  // TM1638 interface
   GPIO_MP3_DFR562,                     // RB-DFR-562, DFPlayer Mini MP3 Player
+  GPIO_MP3_DFR562_BUSY,                // RB-DFR-562, DFPlayer Mini MP3 Player busy flag
```

Comment:
As stated at the top of the file only append this list. never insert as it breaks all devices.